### PR TITLE
First release

### PR DESCRIPTION
**Pull request by QuentinPerez**
Works fine now :)

### CODE REVIEW

**Comment by Kloadut** [comment]
Good job!

Happy to see that it works well on Vivid :-)

**Comment by moul** [comment]
LGTM, let's build the first release and flag it beta :)

File: patches/etc/supervisor/conf.d/cozy-controller.conf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 autorestart=true
 autostart=true
 command=cozy-controller
-environment=NODE_ENV=%(ENV_NODE_ENV)s
+environment=NODE_ENV=productions
```
Comment (nono) [inline patches/etc/supervisor/conf.d/cozy-controller.conf]: I think it should be `production`, with no 's' at the end.

Comment (Kloadut) [inline patches/etc/supervisor/conf.d/cozy-controller.conf]: +1

Comment (QuentinPerez) [inline patches/etc/supervisor/conf.d/cozy-controller.conf]: Fixed https://github.com/scaleway-community/scaleway-cozy/commit/678457054d64828c0d0f014e7ed268cf553fa87a thanks you :)

**Pull request merged**